pyproject.toml

```diff
@@ -38,7 +38,7 @@ classifiers=[
 ]
 dependencies = [
     "torch",
-    "transformer-engine@git+https://github.com/NVIDIA/TransformerEngine.git@v0.7#egg=transformer-engine",
+    "transformer-engine@git+https://github.com/NVIDIA/TransformerEngine.git@v0.11#egg=transformer-engine",
```

Comment:
I have a question on it. There is no version requirement like `transformer-engine==v0.11`. Will TransformerEngine be re-install when installing MS-AMP even if the version of local TransformerEngine is v0.11?

Reply:
I believe TE will not be re-installed if there is already a same version. Just as below:
![image](https://github.com/Azure/MS-AMP/assets/4133758/ff7d221a-07e3-4a07-b505-eb5343e27353)